lenovo/thinkpad/x230/default.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/hdd
```

Comment:
My ThinkPad X230 has a 128GB SSD. I inherit HDD/SSD profiles only when hard drive is soldered in, or proprietary, or otherwise it can be determined that there can only be this particular kind of storage.

You can import this profile in `/etc/nixos/configuration.nix`, though.

Reply:
Also see https://github.com/NixOS/nixpkgs/issues/33033.

Reply:
actually i've copy-pasted this from the x220 profile but i was unsure myself if i really need this in the profile. i remove the import

Reply:
I've left it there for reverse compatibility, I'll start removing things like these soon, but I'll have to ping everyone who has ever contributed to X220 profile, because it's a breaking change. There are also a few places that explicitly specify boot loader to use, which I'm also going to drop.